pywbem/_subscription_manager.py

```diff
@@ -541,6 +557,22 @@ def add_listener_destinations(self, server_id, listener_urls, owned=True):
             permanent. See :ref:`WBEMSubscriptionManager` for details about
             these ownership types.
 
+          persistence_type (:class:`py:int` or None):
+            The PersistenceType property defines for the WBEM server
+            persistence of the destination instance as follows: 1-Other, 2-
+            Permanent, 3-Transient. Most WBEM servers set the value to
+            2(Permanent) if no value is provided with a new instance. pywbem
+            does not support the Other value because it does not support the
+            OtherPersistenceType property.
+
+            If persistence_type is None, the PersistenceType property is
+            created with value 3(Transient) for owned destinations; it is not
+            created for permanent destinations.
+
+            If persistence_type is a valid integer value, the PersistenceType
+            property is created with that value in for both owned and permanent
+            destinations.
```

Comment:
Too complicated. We already say earlier that this sets the PersistenceType property, and there is no need to make the statement about valid values, because invalid values raise an exception anyway. The only ifnromation left to say is that this parameter applies to both owned and permanent destinations.

Reply:
No.  If the user does not supply the parameter, the default value is None. With value None, No PersistenceType property is created for permanent destinations but the PersistenceType property value is 3(Transient) for owned destinations.

Changed the text to clarify.